rossum/user.py

```diff
@@ -110,3 +111,14 @@ def change_command(
 def delete_command(ctx: click.Context, id_: int) -> None:
     with RossumClient(context=ctx.obj) as rossum:
         rossum.patch(f"{USERS}/{id_}", {"is_active": False})
+
+
+def change_password(rossum: RossumClient, id_: int, password: str):
+    if id_ == rossum.get_user().get("id"):
+        rossum.change_user_password(password)
+        return click.echo(
+            "Password has been successfully changed.\n"
+            'Please exit and run "rossum configure" to update existing credentials.'
+        )
+    else:
```

Comment:
Let's substitute `else` with resetting other user's password. You can create a new function for resetting password in `api_client.py`, checking the email of the username with the particular `id` and calling the `/reset` endpoint to send the user the email.